ci/bump_version.sh

```diff
@@ -1,12 +1,32 @@
 #!/bin/bash
 
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+MAGENTA='\033[0;35m'
+CYAN='\033[0;36m'
+GRAY='\033[0;37m'
+
+# Reset
+NC='\033[0m'
+
 if [ $# != 1 ]
 then
      echo "Please provide version as the first argument"
      exit 1
 fi
 
 version="$1"
+regex="^[0-9]+\.[0-9]+\.[0-9]+$"
+
+if [[ $version =~ $regex ]]; then
+  echo -e "${CYAN}Changing to the version number $version${NC}"
```

Comment:
What does the `-e` mean for this command? I couldn't find it. I have no idea what it really is but I thought maybe something with an error? Maybe not... But you should know when you use it. 

If it is something about errors this message doesn't seem to be intended as an error...

Reply:
This `-e` is required to print the text using different colours.